impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -445,13 +452,26 @@ void TextureGLES::InitializeContentsIfNecessary() const {
       {
         TRACE_EVENT0("impeller", "RenderBufferStorageInitialization");
         if (type_ == Type::kRenderBufferMultisampled) {
-          gl.RenderbufferStorageMultisampleEXT(
-              GL_RENDERBUFFER,               // target
-              4,                             // samples
-              render_buffer_format.value(),  // internal format
-              size.width,                    // width
-              size.height                    // height
-          );
+          // BEWARE: these functions are not at all equivalent! the extensions
+          // are from EXT_multisampled_render_to_texture and cannot be used
+          // with regular GLES 3.0 multisampled renderbuffers/textures.
+          if (gl.GetCapabilities()->SupportsImplicitResolvingMSAA()) {
+            gl.RenderbufferStorageMultisampleEXT(
+                GL_RENDERBUFFER,               // target
```

Comment:
please use proper argument comments to get them checked by the linter (ex. /\*foobar=\*/)

Reply:
doesn't look like the linter is checking these, maybe something to do with the proc table drop argument names? either that or its a C ABI.

will switch the comment stye regardless

Reply:
Weird, I double checked we do have `bugprone-argument-comment` in `.clang-tidy`.